target_snooping:
  ros__parameters:

    tilt_static: 0.05
    discretization: 3
    time_to_sleep: 3.0

    target_type: aruco
    marker_id: 20

    subscribers:
      marker_in_sight_prefix: /target_tracking/camera_to_marker_presence/marker_
      
    services:
      set_tilt_static: /target_snooping/set_tilt_static

    actions: 
      start: /target_snooping/start_action

    services_client:
      set_pan_tilt: /ptu/set_pan_tilt
      set_pan_tilt_speed: /ptu/set_pan_tilt_speed
      get_limits: /ptu/get_limits

    limits:
      mode: manual
      pan_min: -0.3
      pan_max: 0.3